tests/cephfs/cephfs_mirroring/test_cephfs_mirror_disconnect.py

```diff
@@ -0,0 +1,461 @@
+            target_clients[0],
+            "client.admin",
+            subvol1_path,
+            source_clients[0],
+            kernel_mounting_dir_1,
+            target_fs,
+        )
+        if not out:
+            raise CommandFailed(f"{rc}")
+        log.info("=" * 50)
+        log.info(
+            "Sceneario 2 : Disconnect the mirroring by removing the peer connection."
+        )
+        log.info("=" * 50)
+        source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"mkdir -p {fuse_mounting_dir_1}/{subvol2_path}/{mounting_dir};"
+            f"python3 /home/cephuser/smallfile/smallfile_cli.py "
+            f"--operation create --threads 1 --file-size 1 "
+            f"--files 5 --files-per-dir 1 --dirs-per-dir 2 --top "
+            f"{fuse_mounting_dir_1}/{subvol2_path}/{mounting_dir}",
+        )
+        peer_uuid = fs_mirroring_utils.get_peer_uuid_by_name(
+            source_clients[0], source_fs
+        )
+        source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"mkdir {fuse_mounting_dir_1}{subvol2_path}.snap/{snap2}",
+        )
+        fs_mirroring_utils.remove_snapshot_mirror_peer(
+            source_clients[0], source_fs, peer_uuid
+        )
+        out, _ = source_clients[0].exec_command(
+            f"ceph fs snapshot mirror peer_list {source_fs} --format json-pretty"
+        )
+        log.info(out)
+        if out:
+            log.error("Peer list not removed")
+        log.info("Import token again")
+        log.info(token)
+        fs_mirroring_utils.import_peer_bootstrap(source_fs, token, source_clients[0])
+        max_duration_seconds = 5 * 60
+        start_time = time.time()
+        while True:
+            peer_uuid_new = fs_mirroring_utils.get_peer_uuid_by_name(
+                source_clients[0], source_fs
+            )
+            if peer_uuid == peer_uuid_new:
+                log.info("Peer UUID has not changed even after new import ")
+                time.sleep(30)
+
+                # Check if the maximum duration has been reached
+                elapsed_time = time.time() - start_time
+                if elapsed_time >= max_duration_seconds:
+                    log.info("Maximum duration reached. Breaking out of the loop.")
+                    break
+            else:
+                break
+
+        fs_mirroring_utils.validate_snapshot_sync_status(
+            cephfs_mirror_node,
+            source_fs,
+            snap2,
+            fsid,
+            asok_files,
+            filesystem_id,
+            peer_uuid_new,
+        )
+        peer_uuid = peer_uuid_new
+        out, rc = fs_mirroring_utils.list_and_verify_remote_snapshots_and_data_checksum(
+            target_clients[0],
+            "client.admin",
+            subvol2_path,
+            source_clients[0],
+            fuse_mounting_dir_1,
+            target_fs,
+        )
+        if not out:
+            raise CommandFailed(f"{rc}")
```

Comment:
Same as above, please print a message that snap2 was synced after peer remove/add workflow